src/writer.jl

```diff
@@ -22,17 +22,23 @@ function write(path::AbstractString, obj; force=false)
         end
     end
 
-    # Handle tabular data
     if Tables.istable(obj)
-        geomcol = first(GI.geometrycolumns(obj))
-        geoms = Tables.getcolumn(obj, geomcol)
-        @warn "DBFTables.jl does not yet `write`, so only .shp, .shx, and .prj files can be written."
-        # DBFTables.Table(obj) # TODO remove geom column
-        # DBFTables.write # TODO DBF write function
+        sch = Tables.schema(obj)
+        tbl = Dict(field => Tables.getcolumn(obj, field) for field in sch.names)
+        # find first column that only contains geometries (and possibly missing)
+        geomcol = findfirst(tbl) do data
```

Comment:
The geomcol = first(GI.geometrycolumns(obj)) is the idiomatic way to check this, where geometrycolumns defaults to a `(:geometry,)`, but one should indeed check isgeometry afterwards.

Besides, there *could* be multiple geometry columns, we only take the first, but we should pop the remainder as well, because I don't expect DBF to like them much.

Reply:
DBF will hate them

Reply:
> The geomcol = first(GI.geometrycolumns(obj)) is the idiomatic way to check this

But say for DataFrames, this will just return `:geometry` every time.  Isn't it better to check the column types?

Reply:
That's a good point.

Can we first check the columnnanes, and check the types as a fallback?

Reply:
That leaves an edge case that will error if you have a `:geometry` column that doesn't contain geometries, but I'll defer to y'all.